docker-integrations/tendermint/src/proto/runtime.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
```

Comment:
@kchrist-rocketfueldev Do we need the `import` here? We are not utilizing `message Empty {}` from `google/protobuf/empty.proto` so can we safely remove it?

Reply:
no we don't it is already in my todo list to remove